api/base/serializers.py

```diff
@@ -606,66 +595,53 @@ class RelationshipField(ser.HyperlinkedIdentityField):
     json_api_link = True  # serializes to a links object
 
     def __init__(
-        self, related_view=None, related_view_kwargs=None, self_view=None, self_view_kwargs=None,
-        self_meta=None, related_meta=None, always_embed=False, filter=None, filter_key=None, required=False, **kwargs
+        self, *,
```

Comment:
What does this bare asterisk do in the parameter list? Does it collect positional args and then not do anything with them?

Reply:
it puts a cap on positional args -- all the params after `*` *must* be passed as keyword args (the ones before may be positional or keyword)
```
>>> def foo(one, *, two):
...     print(f'{one}, {two}')
... 
>>> foo(1, two=2)
1, 2
>>> foo(one=1, two=2)
1, 2
>>> foo(1, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() takes 1 positional argument but 2 were given
```
it gave me comfort in a change like this to guarantee there are no calls anywhere which depend on the order params are listed here